pandas/io/excel/_odswriter.py

```diff
@@ -189,15 +189,15 @@ def _make_table_cell(self, cell) -> tuple[object, Any]:
             value = str(val).lower()
             pvalue = str(val).upper()
         if isinstance(val, datetime.datetime):
-            value = val.isoformat()
-            pvalue = val.strftime("%c")
+            value = val.isoformat()  # fast formatting
```

Comment:
don't use line comments, instead follow the existing style, e.g. comments before